src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -254,7 +254,7 @@ class GitHubEnterpriseIntegrationProvider(GitHubIntegrationProvider):
     name = "GitHub Enterprise"
     metadata = metadata
     integration_cls = GitHubEnterpriseIntegration
-    has_stacktrace_linking = False
+    features = frozenset([IntegrationFeatures.COMMITS, IntegrationFeatures.ISSUE_BASIC])
```

Comment:
GHE has to explicitly define this now because it does _not_ have stack trace linking and can't just inherit the features from GH